packages/shared/styles/classes/_mixins.scss

```diff
@@ -0,0 +1,25 @@
+@mixin color-modifiers($prefix: '-', $property: 'color') {
+  @each $name, $hex in $colors {
+    &#{$prefix}#{$name} {
+      #{$property}: $hex;
+    }
+  }
+}
+
+@mixin space-modifiers($prefix: '-', $property: 'padding') {
```

Comment:
isn't that the same as -multiple but with 1 argument? for readability maybe we should have all modifiers as multiple (but named as this one)